src/metaswitch/crest/settings.py

```diff
@@ -100,7 +100,7 @@ def ensure_dir_exists(d):
 CYCLONE_DEBUG = False  # Make cyclone emit debug messages to the browser etc.
 
 # Include any locally-defined settings.
-_local_settings_file = os.path.join(_MY_DIR, "local_settings.py")
+_local_settings_file = os.environ.get('CREST_SETTINGS', os.path.join(_MY_DIR, "local_settings.py"))
```

Comment:
After discussion with @rkd-msw we think it would be better for this to be a command line argument to the `crest` server process (set up in the homer/homestead-prov `init.d` script).  This is a new approach for crest but there's precedent in some of our other python processes.